Temporarily suppress console errors on Job unit tests.

willianrampazzo opened the pull request:
When running selftests, two unit tests from the Job class are throwing `No test references provided nor any other arguments resolved into tests. Please double check the executed command.`
error message to the output. The messages were suppressed by this change when the run method from the Job class is called to avoid polluting the output.

Signed-off-by: Willian Rampazzo <willianr@redhat.com>

clebergnu commented:
I guess you're referring to the `No test references provided nor any other arguments resolved into tests. Please double check the executed command.` message, right? Would be nice to mention it.

clebergnu commented on selftests/unit/test_job.py:
```diff
@@ -204,7 +205,10 @@ def test_job_run_account_time(self):
         self.job = job.Job(config)
         self.job.setup()
+        # temporarily disable logging on console
+        logging.disable(logging.ERROR)
```
One thing to consider here is that `show` is set to `['none']`, so in theory Avocado should not be producing any output. Maybe the best fix is to suppress it on the job class itself.

willianrampazzo replied:
That is an interesting point I have not noticed.

clebergnu replied:
Actually, I take it back, because of:

https://github.com/avocado-framework/avocado/blob/master/avocado/core/output.py#L46

It would require changing the semantics, so I guess your simpler approach is preferable at this point.

willianrampazzo commented:
> I guess you're referring to the `No test references provided nor any other arguments resolved into tests. Please double check the executed command.` message, right? Would be nice to mention it.

Okay, updated the PR description reflecting the commit message that was also updated.
